x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/optimizer/PhysicalPlanOptimizerTests.java

```diff
@@ -1884,6 +1915,17 @@ private PhysicalPlan optimizedPlan(PhysicalPlan plan, SearchStats searchStats) {
             return EstimatesRowSize.estimateRowSize(fragment.estimatedRowSize(), localPlan);
         });
 
+        // handle local reduction alignment
```

Comment:
This block of code is identical to the one in [LocalPhysicalPlanOptimizerTests.java](https://github.com/elastic/elasticsearch/pull/100866/files#diff-89f6975cd4e2ea37809dd0ccdcc513c4814f5362b503cd1f6f6a492fffeaaa3f).